extensions/connect-to-vpn/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "connect-to-vpn",
+  "title": "Connect to VPN",
+  "description": "Connect to/disconnect from VPN",
+  "icon": "command-icon.png",
+  "author": "sato11",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show network services",
```

Comment:
```suggestion
      "title": "Show Network Services",
      "subtitle": "VPN",
```

Reply:
Can't figure out the right naming, it doesn't taste right 😄. We do need the subtitle at least

Reply:
Yea it is hard in fact. Borrowed the term from the command line option. In my machines the "network services" consist only of VPNs plus stuff like wifi and bluetooth (which I denylisted), but I bet there must be a bunch of other stuff in the wild. It would have been fairly better if I could get only VPNs, but `networksetup -listpppoeservices`, which seems like the one, did not list my VPN 😢